Makefile

```diff
@@ -32,6 +32,11 @@ export GOPROXY
 # Active module mode, as we use go modules to manage dependencies
 export GO111MODULE=on
 
+# Kubebuilder.
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.23.3
+export KUBEBUILDER_CONTROLPLANE_START_TIMEOUT ?= 60s
```

Comment:
Could you clarify on where `KUBEBUILDER_CONTROLPLANE_START_TIMEOUT` and `KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT` get used? And where are we getting the 60s value from?

Reply:
Good question, it looks like this is referenced from https://github.com/kubernetes-sigs/controller-runtime/blob/11917e9d90625b51e5dab16b4cf20934971e2f08/pkg/envtest/server.go#L52 . 

I used the same timeout that cluster-api was using so we can definitely tune this.  Do you think this is what could be causing the build failures?

Reply:
It's definitely possible. The CAPZ test job is giving an error like this. I'm not too familiar with how the bins get built, but something you could try is to figure out what the timeout values are being set to on main and adjust it to match in this PR. If I had to guess I'd say it'd be some default value.

```
[AfterSuite] AfterSuite 
/home/prow/go/src/sigs.k8s.io/cluster-api-provider-azure/exp/controllers/suite_test.go:97
  Expected success, but got an error:
      <errors.aggregate | len:1, cap:1>: [
          <*errors.errorString | 0xc0003ed040>{
              s: "timeout waiting for process kube-apiserver to stop",
          },
      ]
      timeout waiting for process kube-apiserver to stop
  /home/prow/go/src/sigs.k8s.io/cluster-api-provider-azure/exp/controllers/suite_test.go:100
```